models/maat/inst/generate_maat_met_drivers_ngee_pa-bar.R

```diff
@@ -0,0 +1,126 @@
+####################################################################################################
+#
+#   Generate MAAT drivers using NGEE-Tropics QA/QC PA-Bar (BCI) met drivers.  Source format
+#   differs from PA-PNM and PA-SLZ, so for now using two separate met prep scripts
+#   Goal: Get NGEE to provide met drivers in a standard format like AMF to allow for
+#         one met script
+#
+#
+#    --- Last updated: 10.03.2018 By Shawn P. Serbin <sserbin@bnl.gov>
+####################################################################################################
+
+
+#---------------- Close all devices and delete all variables. -------------------------------------#
+rm(list=ls(all=TRUE))   # clear workspace
+graphics.off()          # close any open graphics
+closeAllConnections()   # close any open connections to files
```

Comment:
No change necessary since this just a side script, but Jenny Bryan has a nice [blog post](https://www.tidyverse.org/articles/2017/12/workflow-vs-script/) about why `rm(list = ls())` is a bad idea. But that's just, like, my opinion, man.